bears/css/StyleLintBear.py

```diff
@@ -0,0 +1,32 @@
+import os
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='stylelint',
+        output_format='regex',
+        output_regex=r'.+\n\s(?P<line>\d+):(?P<column>\d+)'
+                     r'\s+✖\s(?P<message>.+)')
```

Comment:
@dracarys09 Why don't you install it the same way in CI you did locally, https://github.com/coala/coala-bears/blob/master/.ci/deps.sh#L101? Did you try that?